torch/utils/hipify/hipify_python.py

```diff
@@ -619,7 +613,7 @@ def is_out_of_place(rel_filepath):
     return True
 
 
-# Keep this synchronized with includes/ignores in build_amd.py
+# deprecated
```

Comment:
@malfet These are added to keep BC linter happy. When are we allowed to get rid of them? Is there a deprecation policy written out somewhere?